account_bank_statement_import_txt_xlsx/__manifest__.py

```diff
@@ -1,28 +1,35 @@
-# Copyright 2014-2017 Akretion (http://www.akretion.com).
-# @author Alexis de Lattre <alexis.delattre@akretion.com>
-# @author Sébastien BEAU <sebastien.beau@akretion.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2019 ForgeFlow, S.L.
+# Copyright 2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
 {
     "name": "Account Bank Statement Import TXT XLSX",
-    'summary': 'Import TXT/CSV or XLSX files as Bank Statements in Odoo',
-    "version": "12.0.1.0.1",
+    "summary": "Import TXT/CSV or XLSX files as Bank Statements in Odoo",
+    "version": "12.0.2.0.0",
     "category": "Accounting",
     "website": "https://github.com/OCA/bank-statement-import",
-    "author": " Eficent, Odoo Community Association (OCA)",
+    "author":
+        "Eficent, "
```

Comment:
Can you update name to ForgeFlow here as well? Thanks!

Reply:
Done